<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewerWindow</class>
 <widget class="QMainWindow" name="ViewerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ViewerWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>198</width>
           <height>438</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="geometry" stdset="0">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>148</width>
            <height>394</height>
           </rect>
          </property>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="ODBPPGraphicsMiniMapView" name="miniMapView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>150</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="ODBPPGraphicsView" name="viewWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionSetColor"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
    <addaction name="actionHome"/>
    <addaction name="actionAreaZoom"/>
    <addaction name="separator"/>
    <addaction name="actionPanLeft"/>
    <addaction name="actionPanRight"/>
    <addaction name="actionPanUp"/>
    <addaction name="actionPanDown"/>
    <addaction name="separator"/>
    <addaction name="actionShowNotes"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionHome"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
   <addaction name="actionAreaZoom"/>
   <addaction name="actionMousePan"/>
   <addaction name="separator"/>
   <addaction name="actionHighlight"/>
   <addaction name="actionClearHighlight"/>
   <addaction name="separator"/>
   <addaction name="actionPanLeft"/>
   <addaction name="actionPanRight"/>
   <addaction name="actionPanUp"/>
   <addaction name="actionPanDown"/>
   <addaction name="separator"/>
   <addaction name="actionShowNotes"/>
  </widget>
  <action name="actionSetColor">
   <property name="text">
    <string>Color</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset>
     <normaloff>icons/zoom-in.png</normaloff>icons/zoom-in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset>
     <normaloff>icons/zoom-out.png</normaloff>icons/zoom-out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionHome">
   <property name="icon">
    <iconset>
     <normaloff>icons/zoom-original.png</normaloff>icons/zoom-original.png</iconset>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </action>
  <action name="actionAreaZoom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/gtk-fullscreen.png</normaloff>icons/gtk-fullscreen.png</iconset>
   </property>
   <property name="text">
    <string>Area Zoom</string>
   </property>
  </action>
  <action name="actionPanLeft">
   <property name="icon">
    <iconset>
     <normaloff>icons/back.png</normaloff>icons/back.png</iconset>
   </property>
   <property name="text">
    <string>Pan Left</string>
   </property>
  </action>
  <action name="actionPanRight">
   <property name="icon">
    <iconset>
     <normaloff>icons/forward.png</normaloff>icons/forward.png</iconset>
   </property>
   <property name="text">
    <string>Pan Right</string>
   </property>
  </action>
  <action name="actionPanUp">
   <property name="icon">
    <iconset>
     <normaloff>icons/up.png</normaloff>icons/up.png</iconset>
   </property>
   <property name="text">
    <string>Pan Up</string>
   </property>
  </action>
  <action name="actionPanDown">
   <property name="icon">
    <iconset>
     <normaloff>icons/down.png</normaloff>icons/down.png</iconset>
   </property>
   <property name="text">
    <string>Pan Down</string>
   </property>
  </action>
  <action name="actionHighlight">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/ok.png</normaloff>icons/ok.png</iconset>
   </property>
   <property name="text">
    <string>Highlight</string>
   </property>
  </action>
  <action name="actionClearHighlight">
   <property name="icon">
    <iconset>
     <normaloff>icons/remove.png</normaloff>icons/remove.png</iconset>
   </property>
   <property name="text">
    <string>Clear Highlight</string>
   </property>
   <property name="toolTip">
    <string>Clear hightlight</string>
   </property>
  </action>
  <action name="actionShowNotes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/document-properties.png</normaloff>icons/document-properties.png</iconset>
   </property>
   <property name="text">
    <string>Show Notes</string>
   </property>
  </action>
  <action name="actionMousePan">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/visualization.png</normaloff>icons/visualization.png</iconset>
   </property>
   <property name="text">
    <string>MousePan</string>
   </property>
   <property name="toolTip">
    <string>Mouse panning</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ODBPPGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>odbppgraphicsview.h</header>
  </customwidget>
  <customwidget>
   <class>ODBPPGraphicsMiniMapView</class>
   <extends>QGraphicsView</extends>
   <header>odbppgraphicsminimapview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
